.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- Don't forget to note any issues here with fixes #<issue number> -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+
+## Screenshots (if appropriate):
+
+## Checklist:
+<!--- Please mark an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the **CONTRIBUTING** document.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have added tests to cover my changes.
+- [ ] I have run yarn validate to ensure that tests, typescript and linting are all in order.
```

Comment:
This one isn't really necessary since it gets run in Travis automatically for the PR

Reply:
I think its still nice to have people run the tests though, might encourage more passing prs in our pr queue